index.js

```diff
@@ -52,6 +52,10 @@ const app = async () => {
         await updateMovieName(argv.updatename, argv.newname);
     } else if (argv.updateyear) {
         await updateMovieYear(argv.name, argv.newyear);
+    } else if (argv.deleteone) {
```

Comment:
This is totally fine, but you could modify it slightly such that the command line flags would be either "--delete" or "--delete all" the code would be a little different but the user interface on the terminal might be nicer since it's an extension of the delete command rather than two delete commands

Reply:
I shall edit and update in another branch.

Reply:
Ah, you can commit on top of any pull request you like, you don't have to, but you could